src/Generator.ts

```diff
@@ -67,6 +67,7 @@ export default class Generator {
 
     const arrivalTime = departureTime.plus({ hours: duration.hours, minutes: duration.minutes }).setZone(destination.timezone);
 
+    const randCost = distance * 0.3;
```

Comment:
Can we add a bit more variability here? Maybe make the `0.3` a randomly generated value? This way flights will differ in cost from day to day.

In order to keep the data the same each time it is retrieved, you'll need to use `this.random`.

Reply:
Oh! Yes! This was actually on our list to go back and change, though I never got to it. I can definitely make that change.

Reply:
Updated!